_data/packages.yml

```diff
@@ -355,7 +395,7 @@
     - name: Patrick J. Roddy
       github_username: paddyroddy
   repository_link: https://github.com/astro-informatics/sleplet
-  version_submitted: v1.4.4
+  version_submitted: '[v1.4.4](https://pypi.org/project/sleplet/1.4.4)'
```

Comment:
this might require a converation about how we want to populate version_submitted. people don't always have a tagged version but when they do and provide it, it's nice to have.

Reply:
yup! i hear you. One way to fix this COULD be to add a note to the field that looks something like this:

version_submitted (v1.2.3)
version_submitted_link

Or alternatively we could just ask for a link to a release, tag or pypi version?

version_submitted ([v1.4.4](https://www.github.com/sleplet/1.4.4)

we would want to talk to folks about this but clearly we are getting different responses for this field that we will need to populate with a validator!

Reply:
i'm open to doing the thing that is most useful. on our end - adding a validation step that checks the version and tries to parse out any links might be a good first step. 

A second step would be adding an example to the issue template to encourage consistency